stubs/Modifiers/StubsModifier.swift

```diff
@@ -54,7 +54,7 @@ class StubsModifier {
                 if let jsonObject = readStubFile(at: stubsURL) {
                     do {
                         let modified = try modifyStubDictionary(jsonObject)
-                        saveModified(modified, at: stubsURL)
+                        saveModifiedStubDictionary(modified, at: stubsURL)
```

Comment:
I didn't know what was being saved. So I updated the signature to be more specific as to what was being saved.

Reply:
Agree, this name makes better sense